Budgett/config.json

```diff
@@ -1,5 +1,8 @@
 {
     "prefix": "~",
     "generalChatID": "391063932535701505",
```

Comment:
You might have to add a meme chat ID to send the meme there. Otherwise bot-Narc might sweep it up.

Reply:
Gotta add bot check to Meme Nar